src/json_parser.cc

```diff
@@ -44,6 +44,30 @@ bool JSONParser::Parse(const std::string& content) {
   return true;
 }
 
+bool JSONParser::HasField(std::string_view field) {
```

Comment:
This is needed because we need to distinguish why there is no value. Is it because it's not an object (i.e. string) or is it because it's missing.

The case where *assets* is missing is legitimate, but if it's the wrong type we need to throw.